redisson/src/main/java/org/redisson/RedissonBloomFilter.java

```diff
@@ -218,8 +218,8 @@ private void readConfig(Map<String, String> config) {
                 || config.get("size") == null) {
             throw new IllegalStateException("Bloom filter is not initialized!");
         }
-        size = Long.valueOf(config.get("size"));
-        hashIterations = Integer.valueOf(config.get("hashIterations"));
+        size = Long.parseLong(config.get("size"));
```

Comment:
@molekyla  Long.valueOf is better since it also maintains internal cache for Long objects

Reply:
@mrniko even when it is unboxed?